src/style/main.sass

```diff
@@ -18,3 +18,11 @@
     @extend .govuk-link
   ul
     @extend .govuk-list, .govuk-list--bullet
+
+body
+  margin-top: 0
+
+.below-header
```

Comment:
The SASS file is not the correct place for one-off classes that only make sense in one component. Svelte's scoped CSS is very nice, since it lets us keep styles in the same component where they're relevant